clinica/utils/filemanip.py

```diff
@@ -429,21 +429,13 @@ def read_participant_tsv(tsv_file: str) -> Tuple[List[str], List[str]]:
     )
 
 
-def extract_metadata_from_json(json_file: str, list_keys: List[str]) -> List[str]:
-    """Extract fields from JSON file.
-
-    Parameters
-    ----------
-    json_file: str
-        Path to JSON file from which to extract metadata.
-
-    list_keys: List[str]
-        List of keys to extract.
-
-    Returns
-    -------
-    list_values: List[str]
-        List of values extracted from the JSON file and corresponding to the keys.
+def extract_metadata_from_json(
+    json_file: str,
+    list_keys: List[str],
+    handle_missing_keys: Optional[Callable] = None,
+) -> List[str]:
+    """Same function but with additional parameter.
```

Comment:
I wrote this for the example, you need to keep the previous docstring and update with the description of the added functionality and parameter.